mewarpx/tests/test_package.py

```diff
@@ -0,0 +1,21 @@
+"""Test basic package aspects"""
+from __future__ import print_function
+
```

Comment:
Do we have or plan to have any python2 compatibility? If not this line can be removed, right?
```suggestion
```

Reply:
Nope, that was just a copy and paste leftover, good catch!